AppInspector/rules/default/os/dynamic_execution.json

```diff
@@ -271,5 +271,59 @@
         "confidence": "high"
       }
     ]
+  },
+  {
+    "name": "OS: Dynamic Execution",
+    "id": "AI035520",
+    "description": "OS: Dynamic Execution",
+    "applies_to": [
+      "php"
+    ],
+    "tags": [
+      "OS.Process.DynamicExecution"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "shell_exec",
```

Comment:
```suggestion
        "pattern": " shell_exec[ ]*\\(",
```

Reply:
Explanation:
` ` before the word to ensure its not a part of another word. 
`[ ]*` after the word to allow for formatting like `methodName (arguments)`
`\\(` to capture the opening paren